### DISCUSSION TITLE
Arabic or Persian FONT

### DISCUSSION THREAD

**Discussion by Raven-rvn** [Bodmer/TFT_eSPI#3026] [Q&A - Fonts]
hi.
can anyone help me?
how can i write arabic or persian words ?

microcontroller : ESP32
Screen : TFT 240 * 240 ST7789 Driver

**Comment by Bodmer** [comment]
That is possible. Example here in the TFT_eFEX extension library:

https://github.com/Bodmer/TFT_eFEX/tree/master/examples/drawStringRTLAr

**Reply by Raven-rvn** [reply]
Thank you.
